server/api/region.go

```diff
@@ -130,6 +131,47 @@ func (h *regionsHandler) CheckRegionsReplicated(w http.ResponseWriter, r *http.R
 	h.rd.JSON(w, http.StatusOK, state)
 }
 
+// @Tags     region
+// @Summary
+// @Accept   json
+// @Param    start_key  body  string  true  "Regions start key, hex encoded"
+// @Param    end_key    body  string  true  "Regions end key, hex encoded"
+// @Param    batch_size  body  string  true  "Maximum operators scheduled in one"
+// @Param    timeout    body  string  true  "Timeout time in milliseconds"
```

Comment:
I think using "seconds" instead of "milliseconds" is more reasonable.
What's more, there is a "timeout" param in the "/admin/unsafe/remove-failed-stores", which unit is "seconds".

Reply:
Fixed